content/landing-page/README.md

```diff
@@ -0,0 +1,9 @@
+---
+slug: /
+---
+
+# Intro
```

Comment:
In the future we should change this content to (parts of) that of the `README.md` file in the OS repo: https://github.com/open-education-hub/operating-systems/blob/master/README.md